pkg/crc/config/callbacks.go

```diff
@@ -19,9 +22,14 @@ func RequiresDeleteMsg(key string, _ interface{}) string {
 }
 
 func RequiresDeleteAndSetupMsg(key string, _ interface{}) string {
-	return fmt.Sprintf("Changes to configuration property '%s' are only applied when the CRC instance is created.\n"+
-		"If you already have a running CRC instance with different %s, then for this configuration change to take effect, "+
-		"delete the CRC instance with 'crc delete', setup it with `crc setup` and start it with 'crc start'.", key, key)
+	// since we cannot easily import the machine package here to check for existence of the CRC vm
+	// we rely on the existence of the machine config file to determine if a VM exists
+	if os.FileExists(filepath.Join(constants.MachineInstanceDir, "crc", "config.json")) {
+		return fmt.Sprintf("Changes to configuration property '%s' are only applied when the CRC instance is created.\n"+
+			"If you already have a running CRC instance with different %s, then for this configuration change to take effect, "+
+			"delete the CRC instance with 'crc delete', setup it with 'crc setup' and start it with 'crc start'.", key, key)
+	}
+	return "To confirm your system is ready, and you have the needed system bundle, please run 'crc setup' before start."
```

Comment:
`before 'crc start'.`?

I think we should have similar wording around `crc setup`/`crc start` in both situations?

```
If you already have a running CRC instance with different %s,
for this configuration change to take effect,
first delete the CRC instance with 'crc delete'. 


Then, to confirm your system is ready,
and you have the needed system bundle,
please run 'crc setup' before start."
```

Reply:
updated